File: library/src/main/java/com/henorek/discharge/solutions/builders/DischargeBuilder.java
package com.henorek.discharge.solutions.builders;

import android.content.Context;

public class DischargeBuilder {

  public static DischargeToastBuilder toast(Context context) {
    return new DischargeToastBuilder(context);
  }

  public static DischargeDialogBuilder dialog(Context context) {
    return new DischargeDialogBuilder(context);
  }
}

File: library/src/main/java/com/henorek/discharge/solutions/builders/DischargeBuilder.kt
package com.henorek.discharge.solutions.builders

import android.content.Context

object DischargeBuilder {

  @JvmStatic fun toast(context: Context): DischargeToastBuilder {
    return DischargeToastBuilder(context)
  }

  @JvmStatic fun dialog(context: Context): DischargeDialogBuilder {
    return DischargeDialogBuilder(context)
  }
}
